src/executor/executor_context.cpp

```diff
@@ -43,5 +47,9 @@ type::EphemeralPool *ExecutorContext::GetPool() {
   return pool_.get();
 }
 
+std::string ExecutorContext::GetDatabaseName() const {
```

Comment:
usually this function is set in header file.

Reply:
This is what other functions in the same file do, so gonna keep it like this.